lib/LoginStrategy.types.ts

```diff
@@ -26,6 +26,20 @@ export interface ITokenPayload {
   iss: string;
 }
 
+export interface ISiweMessagePayload {
```

Comment:
I think this can be replaced by the type from the `siwe` package: https://github.com/spruceid/siwe/blob/053f16bf7dce3ff587e00bc4d3754a29ee9c26e6/packages/siwe/lib/client.ts#L20

Reply:
@jrhender That's not a `type`, it is the main `Class`. I looked at their code to see if they have any interface for these attributes.

Reply:
Why you can not use class?
If class is indeed inapropriate you can try siwe.SiweMessage as type or
```
import type {SiweMessage} from 'siwe`;
```